pkg/comp-functions/functions/common/netpol.go

```diff
@@ -8,10 +8,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// TODO: it would probably make sense to call this from the `BootstrapInstanceNs()` function. For future reference...
-
+// CreateNetworkPolicy creates network policy in the instance namespace to allow other namespaces access to the service
 func CreateNetworkPolicy(sourceNs []string, instanceNs string, instance string, svc *runtime.ServiceRuntime) error {
```

Comment:
Line 25 bellow we might want to check if sliNamespace is set? what if a service does not have slos enabled

Reply:
Yeah makes sense, otherwise we add a `""` allow rule, not sure what happens then.